package.json

```diff
@@ -1,19 +1,24 @@
 {
   "name": "escher-suiteapi-js",
-  "version": "3.0.0",
   "description": "Escher Suite Api js",
   "scripts": {
     "test": "NODE_ENV=test mocha --require co-mocha $(find . -name \"*.spec.js\" -not -path \"./node_modules/*\")",
     "code-style": "eslint $(find . -name \"*.js\" -not -path \"./node_modules/*\")",
-    "coverage": "nyc npm test"
+    "coverage": "nyc npm test",
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
   "pre-commit": [
     "code-style"
   ],
+  "release": {
+    "verifyConditions": {
+      "path": "./node_modules/semantic-release/dist/lib/plugin-noop.js"
```

Comment:
don't do this, it can be run accidentally - they now have travis and codeship verifiers